fetch_github_data.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
```

Comment:
add `set -eu -o pipefail` similarly to the other script. This is bash strict mode: strict syntax, no undefined variables and if any command in a pipeline exits with a non-zero exit code it'll cause the whole script to exit with a non-zero exit code. That will help catch a lot of problems. It might cause you some headaches, but it pays off very, very quickly.

Reply:
Will do.